recipes/mutatest/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "mutatest" %}
+{% set version = "2.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c9e6e7084fe63b8595e45d092ad6fcf4a36213e574ef790964126e7f80692d55
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+      - mutatest = mutatest.cli:cli_main
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - coverage >=4.4,<6.0
+    - typing-extensions
+
+test:
+  imports:
+    - mutatest
+  requires:
+    - coverage >=4.4,<6.0
+    - pytest
+    - freezegun
+    - hypothesis
+    - pytest-cov
+  source_files:
+    - pytest.ini
+  commands:
+    - pytest --pyargs mutatest
+
```

Comment:
Can you also add a test for the command-line execution of `mutatest`, e.g. `mutatest -h`, https://conda-forge.readthedocs.io/en/latest/testing.html#command-line-utilities?

Reply:
@npavlovikj thanks for the quick review and the pointer, done in the latest commit and passed.